bokeh/models/annotations.py

```diff
@@ -1019,7 +1021,8 @@ class Whisker(Annotation):
     """)
 
     dimension = Enum(Dimension, default='height', help="""
-    The direction of the band.
+    The direction of the band can be specified by setting this property 
+    to ``height`` (``y`` direction) or ``dimension`` (``x`` direction).
```

Comment:
@cao-jacky as long as we are updating this, I notice it looks like the text has a copy paste error and should be "direction of the *whisker*" would you mind fixing that here as well?

Reply:
Oh oops, will do